src/typeDefs/query.js

```diff
@@ -21,8 +21,10 @@ const query = gql`
             offset: Int = 0,
             "Whether to sort ascending or descending by player first join date."
             order: ListOrder = DESC,
-            "Filter results by last seen player name. Supports SQL LIKE patterns."
-            searchPlayerName: String
+            "If true, searchPlayerName must have all elements match a username, if false any can match."
+            matchAll: Boolean = true,
+            "Filter results by last seen player name. Supports SQL LIKE patterns. Matching parameters defined by matchAll"
+            searchPlayerName: [String]
```

Comment:
This breaks backwards compatibility, correct? I *think* this is fine because we only use this query for players.tallcraft.com. In the future we should consider doing versioned endpoints of the API.

Reply:
While versioning would be a good idea, backwards compatibility actually is not broken by this change. Passing a single string into a string[] field just passes a one-element length array and so it behaves exactly the same with the same old queries. I tested it.